etc/nebula-graphd.conf.production

```diff
@@ -0,0 +1,54 @@
+########## basics ##########
+# Whether to run as a daemon process
+--daemonize=true
+# The file to host the process id
+--pid_file=pids/nebula-graphd.pid
+
+########## logging ##########
+# The directory to host logging files, which must already exists
+--log_dir=logs
+# Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+--minloglevel=0
```

Comment:
Maybe not enable INFO level.

Reply:
On production cluster,  the default log level is INFO.